lib/completely/templates/template.erb

```diff
@@ -4,6 +4,24 @@
 # completely (https://github.com/dannyben/completely)
 # Modifying it manually is not recommended
 
+<%= function_name %>_filter() {
+  local words="$1"
+  local cur=${COMP_WORDS[COMP_CWORD]}
+  local result=()
+
+  if [[ "${cur:0:1}" == "-" ]]; then
+    echo "$words"
+  
+  else
+    for word in $words; do
+      [[ "${word:0:1}" != "-" ]] && result+=("$word")
+    done
+
+    echo "${result[*]}"
+
+  fi
+}
+
 <%= function_name %>() {
```

Comment:
The function at line 7 receives a space-delimited string of possible completions, and:

1. returns the same string if the current word starts with a hyphen
2. returns a new string without any `--flags` if the current word does not start with a hyphen